src/Components/WebAssembly/WebAssembly/test/Hosting/WebAssemblyHostBuilderTest.cs

```diff
@@ -204,7 +204,7 @@ public void Build_AddsConfigurationToServices()
             var host = builder.Build();
 
             // Assert
-            var configuration = host.Services.GetRequiredService<IConfiguration>();
+            var configuration = host.Services.GetRequiredService<WebAssemblyHostConfiguration>();
```

Comment:
Why this change?

Reply:
I changed the DI to inject the `WebAssemblyHostConfiguration` so that users could access the type with the read/write enabled. I'll probably change the DI to register both the `IConfiguration` and `WebAssemblyHostConfiguration` type to have backwards compatibility and expose the new type to users.

Reply:
Do we need to expose the WebAssemblyHostConfiguration  at all?

Reply:
![image](https://user-images.githubusercontent.com/1857993/78931442-cf319580-7a5a-11ea-8792-ff849d4c37d7.png)

After noodling on it, nope.